### PR TITLE
[eloot]v1.6.0 - add support for fossil charm

### PR DESCRIPTION
**Pull request by Deysh**

### CODE REVIEW

File: scripts/eloot.lic [hunk 1]
```diff
@@ -12,12 +12,14 @@
       contributors: SpiffyJr, Athias, Demandred, Tysong, Deysh, Ondreian
               game: Gemstone
               tags: loot
-           version: 1.5.37
+           version: 1.5.38
```
Comment (mrhoribu) [inline scripts/eloot.lic:15]: Due to semantic versioning, this is a new feature and so it should go to 1.6.0.

Comment (mrhoribu) [reply scripts/eloot.lic:15]: Otherwise, works great so far for me.

Comment (Deysh) [reply scripts/eloot.lic]: Version updated to 1.6.0

File: scripts/eloot.lic [hunk 2]
```diff
@@ -2610,20 +2609,24 @@ module ELoot
...
-            ELoot.wait_rt
-            break
-          elsif res =~ /You can only collect/
-            data[location] += ELoot.silver_check - start_silvers
-            ELoot.wait_rt
-            room = Room.current.id
-            ELoot.silver_deposit
-            ELoot.go2(room)
+          unless ELoot.data.charm.nil?
+            res = ELoot.get_res("point ##{ELoot.data.charm.id} at ##{box.id}", /You summon/)
```
Comment (mrhoribu) [inline scripts/eloot.lic:2613]: Functionally the charm works great, but due to splitting it out and not checking the output for the amount of silvers grabbed, it's not updating the silver breakdown chart at the end. Other than that, everything working great!

**Review by mrhoribu — CHANGES_REQUESTED** [review]

File: scripts/eloot.lic (hunk 2 above)
Comment (Deysh) [reply scripts/eloot.lic:2613]: fixed

**Review by mrhoribu — APPROVED** [review]
lgtm

**Pull request merged**